Package.swift

```diff
@@ -197,7 +197,6 @@ extension Array where Element == PackageDescription.SwiftSetting {
 
     result += [
       .enableUpcomingFeature("ExistentialAny"),
-      .enableExperimentalFeature("SuppressedAssociatedTypes"),
```

Comment:
What was the SuppressedAssociatedTypes doing here anyway?

Reply:
The idea was that you could define an `AttachableContainer` that wrapped a move-only type, but since it's still an experimental feature, I removed it to not bake it into the API before the feature is ready.